vc-verifier/js/src/utils/CredentialUtils.js

```diff
@@ -0,0 +1,16 @@
+import { CredentialVerifierConstants } from "../constant/CredentialVerifierConstants";
+
+export const preProcessVerifiableCredential = (credential) => {
+    if (!credential["proof"]) {
+        throw new Error("Proof document is not available in the received credentials.");
+    }
+    const jsonLdProof = credential["proof"];
```

Comment:
can we name it as `proof` only?
By looking at code around there is no context about JsonLd